README.md

```diff
@@ -180,6 +180,13 @@ changeset is probably closed and we open a new changeset.
 Note that changesets are closed automatically after one hour of inactivity, so we don't have to worry about closing
 them.
 
+# Our Contributors ⭐
+
+<a href="https://github.com/pietervdvn/MapComplete
```

Comment:
There is a newline here, breaking the link and the formatting. This must be fixed before merging this in

Reply:
Apologies from my side, I couldn't find the service's privacy policy. 
I have fixed the formatting in my latest commit.